edgelet/edgelet-core/src/authentication.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use crate::AuthId;
```

Comment:
**For all files**: There is an ordering that we like use statements to be in:

use std::...
_newline_
use external crates (serde / failure / etc)
_newline_
use our own crates (edgelet_utils, for example)
_newline_
use crate::... 

A good example is edgelet-iothub/lib.rs

Reply:
Is it possible to configure `rust fmt` to enforce these rules? I makes some re-ordering even now, but appears not all of them.